google/cloud/bigquery/client.py

```diff
@@ -621,10 +657,13 @@ def get_iam_policy(
         body = {"options": {"requestedPolicyVersion": 1}}
 
         path = "{}:getIamPolicy".format(table.path)
-
-        response = self._call_api(
-            retry, method="POST", path=path, data=body, timeout=timeout,
-        )
+        span_attributes = {"path": path, "body": body}
```

Comment:
`body` is a dict, which is not one of the supported attribute types: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/common/common.md#attributes

Reply:
We can type cast to a str so it can still be viewed in the UI.

Reply:
Do you mean like the following?

```suggestion
        span_attributes = {"path": path, "body": str(body)}
```

Might make sense to use a real JSON encoder.

```suggestion
        span_attributes = {"path": path, "body": json.dumps(body)}
```

Reply:
Body feels like it could be too much data to send as part of a span for most API requests (though probably not in this case). What's the way to decide what should go into an attribute and what shouldn't?

Reply:
It was all mentioned in the design doc. I just went based off of information I felt was important. If you see I that I am missing some fields or some fields need not be added then I'd be happy to make those changes.